### ISSUE TITLE
No preview

### ISSUE THREAD

**Issue by Map1ed** [Cubxity/typstudio#35]
I use Windows 11 23H2, and install it.
however it is repeatedly displaying Compiling but now file output,
I have installed typst in my %PATH%
However, I can use the lsp of vscode to compile the file,
why?

**Comment by johnblommers** [comment]
Yes it does this for macOS and for Linux as well. To kickstart this prerelease you should:
1. Create an empty project directory, say `MyProject`
2. Create file `main.typ` inside that directory.
3. Launch Typstudio
4. File -> Open Project
5. Navigate to  `MyProject`and open that directory
6. Click the folder icon in the left most narrow pane.
7. Click oln `main.typ`
8. Now the preview will appear.
9. It will be empty until you start typing in content in the center pane
10. Typstudio does autosave.
11. You can add more files with the + sign and even edit them but they will not appear in the preview pane.

This is prerelease software and about all you can do is export to PDF which works.
